bin/run-test.sh

```diff
@@ -50,5 +50,3 @@ fi
 
 # Run non-interference tests
 node test/non-interference/http-e2e.js || exit 1
-node test/non-interference/express-e2e.js || exit 1
```

Comment:
We currently have a bug internally tracking the upgrade of these tests. We could also hold off on dropping support/upgrading the dependencies until these tests can be upgraded. I can update the commit message.